deploy/helm/kubecf/templates/sizing.yaml

```
@@ -13,212 +13,58 @@ metadata:
-
-{{- if .Values.features.eirini.enabled }}
-{{- if .Values.sizing.bits.instances }}
-    - type: replace
-      path: /instance_groups/name=bits/instances
-      value: {{.Values.sizing.bits.instances}}
-{{- else if not .Values.high_availability }}
-    - type: replace
-      path: /instance_groups/name=bits/instances
-      value: 1
-{{- end }}
-{{- if .Values.sizing.eirini.instances }}
-    - type: replace
-      path: /instance_groups/name=eirini/instances
-      value: {{.Values.sizing.eirini.instances}}
-{{- else if not .Values.high_availability }}
-    - type: replace
-      path: /instance_groups/name=eirini/instances
-      value: 1
-{{- end }}
-{{- end }}
-
-{{- if .Values.sizing.cc_worker.instances }}
-    - type: replace
-      path: /instance_groups/name=cc-worker/instances
-      value: {{.Values.sizing.cc_worker.instances}}
-{{- else if not .Values.high_availability }}
-    - type: replace
-      path: /instance_groups/name=cc-worker/instances
-      value: 1
-{{- end }}
-
-{{- if .Values.sizing.diego_api.instances }}
-    - type: replace
-      path: /instance_groups/name=diego-api/instances
-      value: {{.Values.sizing.diego_api.instances}}
-{{- else if not .Values.high_availability }}
-    - type: replace
-      path: /instance_groups/name=diego-api/instances
-      value: 1
-{{- end }}
-
-{{- if not .Values.features.eirini.enabled }}
-{{- if .Values.sizing.diego_cell.instances }}
-    - type: replace
-      path: /instance_groups/name=diego-cell/instances
-      value: {{.Values.sizing.diego_cell.instances}}
-{{- else if not .Values.high_availability }}
-    - type: replace
-      path: /instance_groups/name=diego-cell/instances
-      value: 1
-{{- end }}
-{{- end }}
+    {{- $instance_groups := list }}
+
+    {{- /* Core instance groups that always exist */}}
+    {{- $instance_groups = append $instance_groups "adapter" }}
+    {{- $instance_groups = append $instance_groups "api" }}
+    {{- $instance_groups = append $instance_groups "auctioneer" }}
+    {{- $instance_groups = append $instance_groups "cc-worker" }}
+    {{- $instance_groups = append $instance_groups "diego-api" }}
+    {{- $instance_groups = append $instance_groups "log-api" }}
+    {{- $instance_groups = append $instance_groups "nats" }}
+    {{- $instance_groups = append $instance_groups "router" }}
+    {{- $instance_groups = append $instance_groups "routing-api" }}
+    {{- $instance_groups = append $instance_groups "scheduler" }}
+    {{- $instance_groups = append $instance_groups "uaa" }}
+    {{- $instance_groups = append $instance_groups "tcp-router" }}
+
+    {{- /* Autoscaler-only instance groups */}}
+    {{- if .Values.features.autoscaler.enabled }}
+    {{- $instance_groups = append $instance_groups "asapi" }}
+    {{- $instance_groups = append $instance_groups "asactors" }}
+    {{- $instance_groups = append $instance_groups "asmetrics" }}
+    {{- $instance_groups = append $instance_groups "asnozzle" }}
+    {{- end }}{{/* if autoscaler */}}
```

I wonder if this works:

```suggestion
    {{- end /* if autoscaler */}}
```

---

It does not. Comments _must_ start right after the brace, unless (and only if) it's preceded by the whitespace-eating modifier. That is, `{{/*` and `{{- /*` only.